attributes/default.rb

```diff
@@ -185,11 +185,13 @@
 # NVIDIA
 default['cluster']['nvidia']['enabled'] = 'no'
 default['cluster']['nvidia']['driver_version'] = '470.141.03'
-default['cluster']['nvidia']['cuda_version'] = '11.4'
+default['cluster']['nvidia']['cuda_version'] = '11.7'
```

Comment:
Did you check that CUDA 11.7 .1 does not contain any breaking change, according to its changelog?
In any case, may you please link the changelog into the PR description?

Reply:
Link added.